pkgs/development/compilers/gerbil/build.nix

```diff
@@ -1,34 +1,51 @@
-{ stdenv, makeStaticLibraries,
+{ gccStdenv, lib, makeStaticLibraries, callPackage,
```

Comment:
Is `callPackage` needed here?

Reply:
Not anymore. I realize that makeStaticLibraries neither.